mpf/assets/show.py

```diff
@@ -574,7 +575,7 @@ def _remove_delay_handler(self):
     def pause(self):
         """Pause show."""
         self._remove_delay_handler()
-        if self.show_config.events_when_stopped:
+        if self.show_config.events_when_paused:
```

Comment:
I believe this was just an oversight in the original implementation, or maybe `events_when_paused` was never set up.